telegram/request/_httpxrequest.py

```diff
@@ -91,6 +86,20 @@ class HTTPXRequest(BaseRequest):
 
             .. versionchanged:: 20.5
                 Accept ``"2"`` as a valid value.
+        proxy (:obj:`str` | ``httpx.Proxy``, optional): The URL to a proxy server or an
```

Comment:
Looking again at httpx code, the proxy parameter has a much wider definition:
```python
URLTypes = Union["URL", str]
ProxiesTypes = Union[URLTypes, "Proxy", Dict[URLTypes, Union[None, URLTypes, "Proxy"]]]
```

Since ptb doesn't need a dict, consider at least chaning to 
```suggestion
        proxy (:obj:`str` | ``httpx.Proxy`` | ``httpx.URL``, optional): The URL to a proxy server or an
```

Reply:
I can add the URL type, yes. I'll skip the dict types though as discussed in #3844 :)